xbmc/video/VideoInfoScanner.cpp

```diff
@@ -1729,8 +1731,7 @@ namespace VIDEO
   {
     if (pDlgProgress)
     {
-      pDlgProgress->SetLine(1, CVariant{showInfo.m_strTitle});
-      pDlgProgress->SetLine(2, CVariant{20361});
+      pDlgProgress->SetLine(1, CVariant{20361}); // Loading episode details
```

Comment:
We should think of possible ways to decouple this from the GUI, maybe introducing a progress handler that the GUI dialog could derive from (or some other pattern). Doesn't really relate to your change though but since you touched this it would be nice to hear your opinion if you have one on the subject.

Reply:
Yes, this would make sense, in another PR if that’s okay for you.